observability/WFLY-14947-Implement_observability_policy_metrics.adoc

```diff
@@ -0,0 +1,72 @@
+ = Implement Observability Policy for Metrics
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+While WildFly/EAP already supports basic metrics in the base WildFly/EAP system, as well as MicroProfile Metrics in the WildFly MicroProfile configuration and EAP XP, there is an effort in Red Hat Middleware Runtimes to implement a consistent observability policy across all Runtime offerings. In the Java space, the library of choice, and the emerging de facto standard for Java developers is https://micrometer.io[Micrometer].  This RFE will add support for Micrometer to WildFly/EAP and adhere to the policy decided upon by the various stakeholders.
+
+== Issue Metadata
+
+=== Issue
+* https://issues.redhat.com/browse/WFLY-14947
+
+=== Related Issues
+* https://issues.redhat.com/browse/EAP7-1686
+
+=== Dev Contacts
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+* [X] Engineering
+* [X] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [x] Managed domain
+* [x] OpenShift s2i
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+* A single registry for metrics
+* Server and JVM metrics tracked and exposed
+* Availability of metric registry for injection into user applications
+* Endpoint to expose metrics in Prometheus format
+* RBAC support for server/jvm metrics
+* Remove base/vendor prefixes (in accordance with Prometheus standards)
+* This subsystem will replace the existing WildFly Metrics subsystem. Documentation will be provided describing the change.
+* If MicroProfile Metrics is enabled, this subsystem will decline to install itself, as is the current behavior of WildFly Metrics.
```

Comment:
@jasondlee We need migration requirements for the existing WildFly Metrics subsystem.

Either:

1) It continues to work but there is appropriate logic to deal with the case where both it and the Micrometer subsystem are in a non-admin-only server config.
2) It becomes a model-only / admin-only subsystem with a migrate op added to migrate to the new subsystem.

Reply:
I've update the language here a bit, but I'm not entirely sure what more you're looking for if this is insufficient.